### PR TITLE
Adding Azure docs

### PR DESCRIPTION
**Pull request by ChrisChinchilla**
If you could take a look when you can @chaudum that would be great.

On staging here - https://staging.crate.io/install/cloud/azure/

The plan is that I will take the Azure / Windows docs and break it out to separate Windows install docs outside of Azure later.

### CODE REVIEW

File: crate/pages/install/cloud/azure.html [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+logo: /static/images/azure-logo.png
+
+{% extends "install.html" %}
+
+{% block install_content %}
+
+<div class="row">
+  <div class="col-sm-8">
+    <p>
+      Microsoft Azure is the second largest and fastest growing prvidor of Cloud Services in the World. It offers a wide variety of options including Windows servers, Containers, Application images and much more.
```
Comment (chaudum) [inline crate/pages/install/cloud/azure.html]: typo: prvidor

Comment (chaudum) [inline crate/pages/install/cloud/azure.html]: containers and application images - lower case?

File: crate/pages/install/cloud/azure.html [hunk 2]
```diff
@@ -0,0 +1,117 @@
+title: Crate & Microsoft Azure
+description:
```
Comment (chaudum) [inline crate/pages/install/cloud/azure.html]: seo description is missing

File: crate/pages/install/cloud/azure.html [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+      </p>
+      <h3>Create Virtual Machines</h3>
+      <p>
+        Next create virtual machines to act as your Crate nodes. In this tutorial, I chose two low-specification Ubuntu 14.04 servers, but you likely have your own preferred configurations. The important step is number 4, where we need to select the Virtual Network created earlier.
+      </p>
+      <p>
+        <img class="img-responsive" src="/static/images/azure-vn-select.png" />
+      </p>
+      <p>
+        Also at stage 4 we can create the endpoints required for opening the ports Crate needs for inter-node communication, 4200 and 4300. You should end up with something like:
```
Comment (chaudum) [inline crate/pages/install/cloud/azure.html]: inter-node communication is on port `4300` (transport port with custom protocol), the java client is also using that port
`4200` is the HTTP port, which needs to be open for all other crate clients

File: crate/pages/install/cloud/azure.html [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+        To set the Unicast hosts for the Crate cluster we change the default configuration file at <i>/etc/crate/crate.yml</i>.
+      </p>
+
+      <p>
+        Uncomment / add these lines:
+      </p>
+
+      <pre><code>discovery.zen.ping.multicast.enabled: false</code></pre>
+
+      <pre><code>discovery.zen.ping.unicast.hosts:
```
Comment (chaudum) [inline crate/pages/install/cloud/azure.html]: maybe also mention the possibility of DNS discovery

**Comment by ChrisChinchilla** [comment]
Updated accordingly @chaudum

**Comment by ChrisChinchilla** [comment]
OK @chaudum I did some research on 'world'. For some reason, my spell checked never shows in HTML files, even though it's set to :(

Corrected…

**Comment by chaudum** [comment]
ack - please squash and merge :)

**Pull request merged**